src/SamplesApp/SamplesApp.UITests/Microsoft_UI_Xaml_Controls/NumberBoxTests/Given_NumberBox.cs

```diff
@@ -57,7 +57,7 @@ public void UpDownTest()
 			Console.WriteLine("Assert that incrementing after typing in a value validates the text first.");
 			var inputBox = numBox.Descendant().Marked("InputBox");
 			inputBox.FastTap();
-			Thread.Sleep(1);
+			Thread.Sleep(200);
```

Comment:
Could we wait for something like the focused state for the control? (e.g. have a textblock that shows the currently focused control and wait that control)

Reply:
@jeromelaban Fixed in last commit (I didn't add a `TextBlock` to keep the XAML inlined with microsoft-ui-xaml repository).